### PR TITLE
Fix incorrect md file extensions in docs

### PR DESCRIPTION
**Pull request by melindafekete**
## About the changes
As a first step towards a docusaurus upgrade, we need to fix all files with a .md extension that use JSX components.

### CODE REVIEW

File: website/docs/using-unleash/deploy/upgrading-unleash.mdx [hunk 1]
```diff
@@ -154,7 +154,7 @@ Read more in the [API token documentation](../../how-to/how-to-create-api-tokens
 
 ### 2. Configuring Unleash {#2-configuring-unleash}
 
-We have done a lot of changes to the options you can pass in to Unleash. If you are manually configuring Unleash you should have a look on the updated [configuring Unleash documentation](./configuring-unleash.md)
+We have done a lot of changes to the options you can pass in to Unleash. If you are manually configuring Unleash you should have a look on the updated [configuring Unleash documentation](./configuring-unleash)
```
Comment (melindafekete) [inline website/docs/using-unleash/deploy/upgrading-unleash.mdx:157]:
```suggestion
We have done a lot of changes to the options you can pass in to Unleash. If you are manually configuring Unleash you should have a look on the updated [configuring Unleash documentation](./configuring-unleash).
```

File: website/docs/reference/strategy-constraints.mdx [hunk 2]
```diff
@@ -174,7 +174,7 @@ Additionally, you can use negation to get _less than or equal to_ and _greater t
 This section gives a brief overview over to use the client SDKs to interact with strategy constraints. The exact steps will vary depending on which client you are using, so make sure to consult the documentation for your specific client SDK.
 :::
 
-Strategy constraints require [the Unleash Context](../reference/unleash-context) to work. All official [Unleash client SDKs](../reference/sdks/index.md) support the option to pass [dynamic context values](../reference/unleash-context#structure "Unleash Context, section: structure") to the `isEnabled` function (or the SDK's equivalent).
+Strategy constraints require [the Unleash Context](../reference/unleash-context) to work. All official [Unleash client SDKs](../reference/sdks) support the option to pass [dynamic context values](../reference/unleash-context#structure "Unleash Context, section: structure") to the `isEnabled` function (or the SDK's equivalent).
```
Comment (melindafekete) [inline website/docs/reference/strategy-constraints.mdx:177]:
```suggestion
Strategy constraints require [the Unleash Context](../reference/unleash-context) to work. All official [Unleash client SDKs](../reference/sdks) support the option to pass [dynamic context values](../reference/unleash-context#structure) to the `isEnabled` function (or the SDK's equivalent).
```

File: website/docs/reference/integrations/slack.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ title: Slack (deprecated)
 
 :::caution Deprecation notice
 
-This Slack integration is deprecated and will be removed in a future release. We recommend using the new [Slack App](./slack-app.md) integration instead.
+This Slack integration is deprecated and will be removed in a future release. We recommend using the new [Slack App](./slack-app) integration instead.
```
Comment (alvinometric) [inline website/docs/reference/integrations/slack.md:8]: This is for another PR, but it feels strange that `/integration/slack` is the old link. We could probably rename this page to `legacy-slack` and the other one to `/slack`

Comment (melindafekete) [reply website/docs/reference/integrations/slack.md:8]: Yes good point. I want to have a PR that looks at deprecations in general, removing them from the navigation etc. and doing a better job at guiding users towards the new functionality. I'll add this note to that project.

**Review by alvinometric — APPROVED** [review]
That's one big set of changes 🎉
I hope you used a regex

**Pull request merged**